zaza/openstack/utilities/upgrade_utils.py

```diff
@@ -41,8 +41,8 @@
         'aodh', 'barbican', 'ceilometer', 'ceph-fs',
         'ceph-radosgw', 'cinder', 'designate',
         'designate-bind', 'glance', 'gnocchi', 'heat', 'manila',
-        'manila-generic', 'neutron-api', 'neutron-gateway', 'placement',
-        'nova-cloud-controller', 'openstack-dashboard']),
+        'manila-generic', 'neutron-api', 'neutron-gateway', 'ovn-central',
```

Comment:
When does the `ovn-chassis` or `ovn-dedicated-chassis` get upgraded?

Reply:
charms not in this list of principle charms get upgraded at the very end of the process. You're right I'll add the dedicated chassis then, thanks!